mk/ebpf.mk

```diff
@@ -0,0 +1,20 @@
+MERBRIDGE_RELEASE_TAG ?= main-20aa4b03cf256a261d1ebfa3af9f390186bec3ae
+MERBRIDGE_RELEASE_REPO ?= https://github.com/kumahq/merbridge
+TARBALL_NAME ?= all.tar.gz
+
+PROGRAMS = $(KUMA_DIR)/pkg/transparentproxy/ebpf/programs/mb_*
+
+.PHONY: build/ebpf
+build/ebpf: $(PROGRAMS)
+
+$(PROGRAMS):
```

Comment:
one minor detail is that changing the tag doesn't retrigger the download. I think it's fine but it might be worth adding a comment saying you should do `clean/ebpf` if you change the tag.

Reply:
ack

Reply:
addressed